db/new_seeds/scenarios/participants/transfers/fip_to_fip_changing_training_provider.rb

```diff
@@ -17,9 +15,11 @@ def build
 
             Rails.logger.info("seeded transfer of #{participant_profile.full_name} from #{school_from.name} to #{school_to.name} using the new school's training provider")
 
-            FactoryBot.create(:seed_induction_record,
-                              participant_profile:,
-                              induction_programme: induction_programme_to)
+            Induction::TransferToSchoolsProgramme.call(participant_profile:,
+                                                       induction_programme: induction_programme_to,
+                                                       start_date:,
+                                                       email:,
+                                                       mentor_profile:)
```

Comment:
I thought we were trying to avoid using services in scenarios/seeds?

Reply:
Yes but if I don't call the service I would need to fully replicate its code here. 
I think that would make this very dirty.
And if we change the code to transfer a participant, we would need to changed this duplicated code?